README.md

```diff
@@ -1,11 +1,19 @@
-The common API for various docker-related plugins to share their configurations. It covers:
+API plugin, which provides the common shared functionality for various Docker-related plugins.
 
+Summary
+---
+
+* API for managing Docker image and container fingerprints
 * Credentials and location of Docker Registry
 * Credentials and location of Docker Daemon (aka Docker Remote API)
+* <code>ToolInstallation</code> for Docker CLI clients
+* <code>DockerImageExtractor</code> extension point to get Docker image relations from jobs
+* Simple UI referring related image fingerprints in Docker builds
+* etc.
+
+More info is available on the plugin's [Wiki page](https://wiki.jenkins-ci.org/display/JENKINS/Docker+Commons+Plugin)
 
-This allows users to configure one set of endpoint/credentials and use it across all the Docker
-related plugins, thereby keeping configuration more [DRY](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
 
-See [here](src/test/java/org/jenkinsci/plugins/docker/commons/SampleDockerBuilder.java)
-and [here](src/test/java/org/jenkinsci/plugins/docker/commons/SampleDockerRegistryBuilder.java)
-and their corresponding `config.jelly` files for an example of how to use them.
+License
+---
+[MIT License](http://opensource.org/licenses/MIT)
```

Comment:
This should be in LICENCE.txt (which is [recommended](https://help.github.com/articles/open-source-licensing/#where-does-the-license-live-on-my-repository) by github)